patches/pciutils/0001-Add-initial-Managarm-support.patch

```diff
@@ -0,0 +1,147 @@
+From f6c4152a19b2430b491ff9b0c0e16a2b368ca60d Mon Sep 17 00:00:00 2001
+From: Dennis Bonke <admin@dennisbonke.com>
+Date: Wed, 27 Apr 2022 23:11:14 +0200
+Subject: [PATCH] Add initial Managarm support
+
+Signed-off-by: Dennis Bonke <admin@dennisbonke.com>
+---
+ lib/filter.c        |  8 ++++----
+ lib/i386-io-linux.h | 25 +++++++++++++++++++++++++
+ lib/i386-ports.c    | 20 ++++++++++++++++++++
+ lib/sysdep.h        |  6 +++---
+ 4 files changed, 52 insertions(+), 7 deletions(-)
+
+diff --git a/lib/filter.c b/lib/filter.c
+index b881b6b..294c69b 100644
+--- a/lib/filter.c
++++ b/lib/filter.c
+@@ -304,20 +304,20 @@ pci_filter_match_v30(struct pci_filter_v30 *f, struct pci_dev *d)
+ 
+ STATIC_ALIAS(void pci_filter_init(struct pci_access *a, struct pci_filter *f), pci_filter_init_v38(a, f));
+ SYMBOL_VERSION(pci_filter_init_v30, pci_filter_init@LIBPCI_3.0);
+-SYMBOL_VERSION(pci_filter_init_v38, pci_filter_init@LIBPCI_3.3);
++//SYMBOL_VERSION(pci_filter_init_v38, pci_filter_init@LIBPCI_3.3);
```

Comment:
Why is this done?

Reply:
I got strange linker errors related to this, so I removed it. If requested, we could investigate it and fix those instead.